doc/cookbook/source/examples/regression/linear_ridge_regression.rst

```diff
@@ -0,0 +1,49 @@
+=======================
+Linear Ridge Regression
+=======================
+
+A linear ridge regression model can be defined as :math:`y_i = \bf{w}^\top\bf{x_i}` where :math:`y_i` is the predicted value, :math:`\bf{x_i}` is a feature vector and :math:`\bf{w}` is the weight vector. We aim to find the linear function that best explains the data, i.e. minimizes the error or loss function :math:`E(\bf{w})` by finding appropriate :math:`\bf{w}`.
+
+.. math::
+    E({\bf{w}}) = \sum_{i=1}^N(y_i-{\bf w}^\top {\bf x}_i)^2 + \tau||{\bf w}||^2
+
+One can show the solution can be written as:
+
+.. math::
+    {\bf w} = \left(\tau {\bf I}+ \sum_{i=1}^N{\bf x}_i{\bf x}_i^\top\right)^{-1}\left(\sum_{i=1}^N y_i{\bf x}_i\right)
+
+where :math:`N` is the number of training samples.
+
+A bias term, which is the squared empirical error, can also be calculated.
+
+For the special case when :math:`\tau = 0`, a wrapper class is defined over :sgclass:`CLinearRidgeRegression` called :sgclass:`CLeastSquaresRegression`.
```

Comment:
For the special case when :math:`\tau = 0`, a wrapper class :sgclass:`CLeastSquaresRegression` is available.